oak-search/src/test/java/org/apache/jackrabbit/oak/plugins/index/FullTextIndexCommonTest.java

```diff
@@ -45,12 +48,12 @@ public void defaultAnalyzer() throws Exception {
         root.commit();
 
         assertEventually(() -> {
-            assertQuery("//*[jcr:contains(@analyzed_field, 'SUN.JPG')] ", XPATH, Collections.singletonList("/test/a"));
-            assertQuery("//*[jcr:contains(@analyzed_field, 'Sun')] ", XPATH, Collections.singletonList("/test/a"));
-            assertQuery("//*[jcr:contains(@analyzed_field, 'jpg')] ", XPATH, Collections.singletonList("/test/a"));
-            assertQuery("//*[jcr:contains(., 'SUN.jpg')] ", XPATH, Collections.singletonList("/test/a"));
-            assertQuery("//*[jcr:contains(., 'sun')] ", XPATH, Collections.singletonList("/test/a"));
-            assertQuery("//*[jcr:contains(., 'jpg')] ", XPATH, Collections.singletonList("/test/a"));
+            assertQuery("//*[jcr:contains(@analyzed_field, 'SUN.JPG')] ", XPATH, singletonList("/test/a"));
+            assertQuery("//*[jcr:contains(@analyzed_field, 'Sun')] ", XPATH, singletonList("/test/a"));
+            assertQuery("//*[jcr:contains(@analyzed_field, 'jpg')] ", XPATH, singletonList("/test/a"));
+            assertQuery("//*[jcr:contains(., 'SUN.jpg')] ", XPATH, singletonList("/test/a"));
```

Comment:
This test seems to be failing at this assertion in this PR checks here https://ci-builds.apache.org/job/Jackrabbit/job/oak-trunk-pr/job/PR-670/1/testReport/junit/org.apache.jackrabbit.oak.plugins.index.elastic/ElasticFullTextIndexCommonTest/oak_search_elastic___defaultAnalyzer/?cloudbees-analytics-link=scm-reporting%2Ftests%2Ffailed 

Seems like the original issue with the exception
`failed to create query: failed to parse date field [sun] with format [strict_date_optional_time||epoch_millis]: [failed to parse date field [sun] with format [strict_date_optional_time||epoch_millis]] 
`

Reply:
done

Reply:
@nit0906 I did not notice this because the test console output does not include log messages.

I have converted this PR to `draft` and I do not plan to merge it. I have improved a bit the logging in case of failure starting from the work done in https://github.com/apache/jackrabbit-oak/pull/649/files#diff-ac2f41f4e63cb259a78a57e764daed1a88e5a1d225161398ee16cc7db60a4034

I have mainly centralized the message in a new Exception avoiding printing the different information in multiple places.

@nfsantos please review this and incorporate this work in the main OAK-9875 PR. Thanks

Reply:
Ok, I'll do that.